script/bigscreenplayer.js

```diff
@@ -80,6 +83,10 @@ define('bigscreenplayer/bigscreenplayer',
         if (evt.data.duration) {
           DebugTool.keyValue({key: 'duration', value: evt.data.duration});
         }
+
+        if (playerComponent && readyHelper) {
```

Comment:
They have to call BSP.init, so PlayerComponent and readyHelper must always be available?

Reply:
If we don't do this, you get race conditions; this is for example what caused the subtitles bug on Native Devices in iPlayer, due to initialization issues